test/hotspot/jtreg/gc/shenandoah/compiler/TestReferenceCAS.java

```diff
@@ -53,32 +53,6 @@
  * @run main/othervm -Diters=20000 -XX:+UnlockDiagnosticVMOptions -XX:+UnlockExperimentalVMOptions -XX:ShenandoahGCHeuristics=aggressive -XX:+UseShenandoahGC -XX:-UseCompressedOops -XX:TieredStopAtLevel=4  TestReferenceCAS
  */
 
-/*
```

Comment:
This one is actually not a regression test, it is similar to `TestClone` that verifies the generic support for Shenandoah barriers. we should keep it here.